src/DotNetWorker/Http/HttpCookie.cs

```diff
@@ -5,30 +5,66 @@
 
 namespace Microsoft.Azure.Functions.Worker.Http
 {
+    /// <summary>
+    /// Http response cookie object
+    /// </summary>
     public sealed class HttpCookie : IHttpCookie
     {
+        /// <summary>
+        /// Creates a cookie with name and value.
+        /// </summary>
+        /// <param name="name">Cookie name</param>
+        /// <param name="value">Cookie value</param>
         public HttpCookie(string name, string value)
         {
             Name = name ?? throw new ArgumentNullException(nameof(name));
             Value = value ?? throw new ArgumentNullException(nameof(value));
         }
 
+        /// <summary>
```

Comment:
For properties, can we use the `Gets or sets .. ` or appropriate (e.g. `Gets ...` for getter only) pattern?